specification/common-types/resource-management/v4/customermanagedkeys.json

```diff
@@ -0,0 +1,70 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "4.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "encryption": {
+      "type": "object",
+      "description": "(Optional) Discouraged to include in resource definition. Only needed where it is possible to disable platform (AKA infrastructure) encryption. Azure SQL TDE is an example of this. Values are enabled and disabled.",
+      "properties": {
+        "infrastructureEncryption": {
```

Comment:
@jackrichins We don't need to support the infrastructureEncryption parameter. This is why I also exposed the "customerManagedKeyEncryption" type below, so that we (and other teams like us) could use it to define at 'lighter' version of "encryption", without infrastructureEncryption parameter.

Reply:
If this is only for your service, why not remove infrastructureEncryption entirely?

Reply:
Because I'm adding it here as a common type to be used by other services as well, following the [preferred schema](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md)